Mixpanel-swift.podspec

```diff
@@ -10,28 +10,36 @@ Pod::Spec.new do |s|
   s.source       = { :git => 'https://github.com/mixpanel/mixpanel-swift.git',
                      :tag => "v#{s.version}" }
   s.ios.deployment_target = '8.0'
-  s.ios.source_files = 'Mixpanel/*.swift'
   s.ios.resources = ['Mixpanel/**/*.{png,xib,storyboard}']
   s.ios.frameworks = 'UIKit', 'Foundation', 'CoreTelephony'
-  s.ios.pod_target_xcconfig = {
-    'OTHER_SWIFT_FLAGS' => '$(inherited) -D DECIDE'
-  }
+  s.default_subspec = 'Complete'
+  base_source_files = ['Mixpanel/Network.swift', 'Mixpanel/FlushRequest.swift', 'Mixpanel/PrintLogging.swift', 'Mixpanel/FileLogging.swift',
+    'Mixpanel/Logger.swift', 'Mixpanel/JSONHandler.swift', 'Mixpanel/Error.swift', 'Mixpanel/AutomaticProperties.swift',
+    'Mixpanel/Constants.swift', 'Mixpanel/MixpanelType.swift', 'Mixpanel/Mixpanel.swift', 'Mixpanel/MixpanelInstance.swift',
+    'Mixpanel/Persistence.swift', 'Mixpanel/Flush.swift','Mixpanel/Track.swift', 'Mixpanel/People.swift', 'Mixpanel/AutomaticEvents.swift']
   s.tvos.deployment_target = '9.0'
-  s.tvos.source_files = ['Mixpanel/Network.swift', 'Mixpanel/FlushRequest.swift', 'Mixpanel/PrintLogging.swift', 'Mixpanel/FileLogging.swift',
-      'Mixpanel/Logger.swift', 'Mixpanel/JSONHandler.swift', 'Mixpanel/Error.swift', 'Mixpanel/AutomaticProperties.swift',
-      'Mixpanel/Constants.swift', 'Mixpanel/MixpanelType.swift', 'Mixpanel/Mixpanel.swift', 'Mixpanel/MixpanelInstance.swift',
-      'Mixpanel/Persistence.swift', 'Mixpanel/Flush.swift','Mixpanel/Track.swift', 'Mixpanel/People.swift', 'Mixpanel/AutomaticEvents.swift']
   s.tvos.frameworks = 'UIKit', 'Foundation'
   s.tvos.pod_target_xcconfig = {
     'OTHER_SWIFT_FLAGS' => '$(inherited) -D TV_OS'
   }
   s.osx.deployment_target = '10.10'
-  s.osx.source_files = ['Mixpanel/Network.swift', 'Mixpanel/FlushRequest.swift', 'Mixpanel/PrintLogging.swift', 'Mixpanel/FileLogging.swift',
-      'Mixpanel/Logger.swift', 'Mixpanel/JSONHandler.swift', 'Mixpanel/Error.swift', 'Mixpanel/AutomaticProperties.swift',
-      'Mixpanel/Constants.swift', 'Mixpanel/MixpanelType.swift', 'Mixpanel/Mixpanel.swift', 'Mixpanel/MixpanelInstance.swift',
-      'Mixpanel/Persistence.swift', 'Mixpanel/Flush.swift','Mixpanel/Track.swift', 'Mixpanel/People.swift', 'Mixpanel/AutomaticEvents.swift']
   s.osx.frameworks = 'Cocoa', 'Foundation'
   s.osx.pod_target_xcconfig = {
     'OTHER_SWIFT_FLAGS' => '$(inherited) -D MAC_OS'
   }
+
+  s.subspec 'Complete' do |ss|
+    ss.ios.pod_target_xcconfig = {
+      'OTHER_SWIFT_FLAGS' => '$(inherited) -D DECIDE'
+    }
+    ss.ios.source_files = 'Mixpanel/*.swift'
+    ss.tvos.source_files = base_source_files
```

Comment:
maybe leave this and osx in the default part of the spec since they're the same between the two

Reply:
I wanted to originally, but pod lib lint complained that the subspec doesn't have source_files :/